libpod/runtime_ctr.go

```diff
@@ -32,28 +31,22 @@ type CtrCreateOption func(*Container) error
 type ContainerFilter func(*Container) bool
 
 // NewContainer creates a new container from a given OCI config
-func (r *Runtime) NewContainer(ctx context.Context, rSpec *spec.Spec, options ...CtrCreateOption) (c *Container, err error) {
+func (r *Runtime) NewContainer(ctx context.Context, options ...CtrCreateOption) (c *Container, err error) {
```

Comment:
Woah woah woah why are we changing this signature

Reply:
We have a chicken and egg situation.  I need the have a container so I can set the labels in the spec file.  I knew you would not like this, but I have no better idea.

Reply:
Put it in `generateSpec()` - store the SELinux config in database Config, then generate the label as part of `generateSpec()` as we generate the final spec. Optionally store the label we're using in database state.

Reply:
Some of the labels are used in CreateConfigToOCISpec.  It needs the mountslabel to label handle anyone doing a 

podman run -v /var/lib/foo:/var/lib/foo:Z

Reply:
Can we get the labels back out of container state after creation succeeds and then do the :Z chcon before we start the container?